src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.Unix.targets

```diff
@@ -50,6 +50,9 @@ The .NET Foundation licenses this file to you under the MIT license.
 
       <EventPipeName>libeventpipe-disabled</EventPipeName>
       <EventPipeName Condition="'$(EventSourceSupport)' == 'true'">libeventpipe-enabled</EventPipeName>
+
+      <LinkStandardCPlusPlusLibrary Condition="'$(LinkStandardCPlusPlusLibrary)' == '' and '$(NativeLib)' != '' and '$(_IsiOSLikePlatform)' == 'true' and '$(InvariantGlobalization)' != 'true'">true</LinkStandardCPlusPlusLibrary>
```

Comment:
Is `'$(NativeLib)' != '' and` necessary and logically correct here? 

Xamarin link targets may be adding the c++ library reference today, but it is not right for runtime to depend on it. The runtime build targets should be coherent.

Reply:
> Is '$(NativeLib)' != '' and necessary and logically correct here?

It is not need and I will remove it, thank you.

> Xamarin link targets may be adding the c++ library reference today, but it is not right for runtime to depend on it. The runtime build targets should be coherent.

From my understanding the Xamarin build uses `clang++` as the native compiler/linker and does not explicitly include the c++ library.